### DISCUSSION TITLE
Accesso selettivo alle segnalazioni - Dlgs 24/2023

### DISCUSSION THREAD

**Discussion by cienne1** [globaleaks/globaleaks-whistleblowing-software#3578] [General]
E' prevista la possibilità di consentire un accesso selettivo alle segnalazioni, abilitando alcuni soggetti/uffici a visualizzare solo alcune segnalazioni o solo alcune parti di esse per finalità istruttorie senza che possano accedere a tutte le altre segnalazioni?
Al momento chi è in possesso di pw di accesso puo' vedere tutte le segnalazioni inviate, anche quelle che potrebbero non dover vedere.

**Comment by mauriziobulgarini** [comment]
Credo che per ottenere questo risultato si possa agire in diversi modi:
1. la soluzione più semplice penso sia quella di usare canali separati per gestire correttamente il routing delle segnalazioni. In questo modo anche chi segnala potrà scegliere il canale corretto in funzione della sua segnalazione, che sarà poi mandata solo a chi di dovere.
2. in alternativa si può dare la possibilità al segnalante di scegliere il/i propri riceventi (l'opzione si attiva dalle impostazioni avanzate del canale - "Abilita i segnalanti a selezionare i propri destinatari")
3. un'alternativa (più laboriosa, ma forse più raffinata) può essere quella di configurare degli utenti con il ruolo di riceventi, ma non associarli a nessun canale. Dopodichè si elegge almeno un ricevente che invece verrà associato ai canali (e quindi riceverà le notifiche delle nuove segnalazioni e potrà vederle), in modo che possa fare da dispatcher delle segnalazioni ai riceventi più idonei. Per fare questo bisogna abilitare l'opzione "Permetti a questo ricevente di concedere l'accesso alle segnalazioni ad altri utenti" nelle impostazioni dell'utente.
4. tutte queste soluzioni riguardano comunque l'intera segnalazione, per quanto riguarda invece la possibilità di vederne solo una parte, l'unica cosa che mi viene in mente è l'utilizzo del ruolo di custode (che permette di celare le domande che rivelano l'identità e i dati di contatto del segnalante)
Vi trovate? Vi vengono altre idee?
